### DISCUSSION TITLE
Overview of available options

### DISCUSSION THREAD

**Discussion by ede1998** [nix-community/plasma-manager#270] [Q&A] [answered]
Hello everyone!
First of all, thank you for developing this. It looks really nice and I want to use it.

Is there a nicely formatted overview available for the available options like for [NixOS](https://search.nixos.org/options) and [Home-Manager](https://nix-community.github.io/home-manager/options.xhtml)? I'm finding it a bit hard right now to get an idea of the options that I could set.

Apologies if I missed it in the README.

**Comment by HeitorAugustoLN** [comment, marked as answer]
Currently, we don't have a documentation of available options. But we are looking into it. See #192

**Reply by ede1998** [reply]
Thanks for the quick answer. I didn't find the issue, probably because I looked for documentation instead of searchability as a keyword.
